### PR TITLE
Update: Add `hidden` flag to be able to hide rootType tab in Treepicker

### PR DESCRIPTION
**Pull request by vinteo**

### CODE REVIEW

File: src/components/adslotUi/TreePickerPureComponent.js [hunk 1]
```diff
@@ -46,14 +46,15 @@ const TreePickerPureComponent = ({
...
+  const visibleRootTypes = _.filter(rootTypes, ({ hidden }) => !hidden);
 
   return (
     <div className="treepickerpure-component">
 
       <SplitPane dts="treepicker-splitpane-available">
 
         <ul className="nav nav-tabs">
-          {rootTypes.length ? _.map(rootTypes, (rootType) =>
+          {visibleRootTypes.length ? _.map(visibleRootTypes, (rootType) =>
```
Comment (patrick-ivers) [inline src/components/adslotUi/TreePickerPureComponent.js]: Even though 0 is a falsey value in javascript, wouldn't it be better to use `visibleRootTypes.length > 0` or `!== 0`?

Comment (vinteo) [inline src/components/adslotUi/TreePickerPureComponent.js]: Our convention is not to do that, its pretty clear what it is checking

Comment (patrick-ivers) [inline src/components/adslotUi/TreePickerPureComponent.js]: Fair enough.

**Comment by patrick-ivers** [comment]
LGTM ✌️

**Comment by vinteo** [comment]
@patrick-ivers merge?

**Comment by OndrejCholeva** [comment]
LGTM :ghost:

**Pull request merged**